web/react/utils/client.jsx

```diff
@@ -28,16 +28,7 @@ function handleError(methodName, xhr, status, err) {
         msg = 'error in ' + methodName + ' status=' + status + ' statusCode=' + xhr.status + ' err=' + err;
 
         if (xhr.status === 0) {
-            let errorCount = 1;
-            const oldError = ErrorStore.getLastError();
-            let connectError = 'There appears to be a problem with your internet connection';
-
-            if (oldError && oldError.connErrorCount) {
-                errorCount += oldError.connErrorCount;
-                connectError = 'Please check connection, Mattermost unreachable. If issue persists, ask administrator to check WebSocket port.';
-            }
-
-            e = {message: connectError, connErrorCount: errorCount};
+            e = {message: 'There appears to be a problem with your internet connection'};
```

Comment:
@enahum yup...good catch.  I was just moving code around.  Looks like client.jsx wasn't localized.  There seems to be some other error messages missing.  I'll add them with this PR.